src/js/main.js

```diff
@@ -395,37 +400,84 @@ Site.Map = {
     var newX = _this.mapPosition[4] + (translation * Math.cos(angle));
     var newY = _this.mapPosition[5] + (translation * Math.sin(angle));
 
+    // Move the map
+    _this.moveMap(newX,newY);
+
+    // Animate recursevly
+    window.requestAnimationFrame(_this.pan.bind(_this));
+
+  },
+
+  // Check if golocation is available and move the map to that equivalent location
+  geolocation: function() {
+    var _this =  this;
+
+    // THIS IS FOR DEVELOPING WHILE WE GET THE REAL API KEY
```

Comment:
actually. the one we had was globie one, free plan. free plan doesnt return lat long. thats why